sdv/single_table/base.py

```diff
@@ -775,31 +768,51 @@ def _transform_helper(self, data):
 
         return data
 
-    def preprocess(self, data):
-        """Transform the raw data to numerical space.
+    def _validate_cags(self, data):
```

Comment:
Should this function just call the user-facing `validate_cag` function (made for synthetic data)?
https://github.com/sdv-dev/SDV/blob/85b506c11d0e6d9ba93e16f12d27dccd3d80c1a6/sdv/single_table/base.py#L719

Reply:
On second thought, these two functions seem slightly different.